deploygate.gemspec

```diff
@@ -28,14 +30,15 @@ POST_INSTALL_MESSAGE
   spec.add_runtime_dependency 'highline', '~> 2.0'
   spec.add_runtime_dependency 'uuid', '~> 2.3'
   spec.add_runtime_dependency 'gem_update_checker', '~> 0.2'
-  spec.add_runtime_dependency 'activesupport', '~> 4.2'
-  spec.add_runtime_dependency 'i18n', '~> 0.7'
```

Comment:
I understand to delete this line because i18n gem was bundled with after Rails3.